src/my_app/test/helpers/pump_route.dart

```diff
@@ -0,0 +1,15 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:very_good_core/l10n/l10n.dart';
+
+extension PumpRoute on WidgetTester {
+  Future<void> pumpRoute(Route<dynamic> route) {
```

Comment:
I'd prefer not to include code if it's not being actively used within the application.

Reply:
As the template is right now no because there is no routes, just the counter...the use case is testing routes like we talked yesterday

Reply:
I think it'd be confusing to have unused code in the template. Maybe we can incorporate this code into other templates where there's a use-case? Wdyt?

Reply:
I can't think on a template where this could be added, this is more useful for an application template.

Any template that comes toyour mind?